qiskit/circuit/library/n_local/qaoa_ansatz.py

```diff
@@ -11,12 +11,15 @@
 # that they have been altered from the originals.
 
 """A generalized QAOA quantum circuit with a support of custom initial states and mixers."""
+
 # pylint: disable=cyclic-import
-from typing import Optional, Set, List, Tuple
+from typing import Optional, List, Tuple
+import numpy as np
 
-from qiskit.circuit.library.evolved_operator_ansatz import EvolvedOperatorAnsatz
-from qiskit.circuit.parameter import Parameter
+from qiskit.circuit.library.evolved_operator_ansatz import EvolvedOperatorAnsatz, _is_pauli_identity
```

Comment:
Yeah that might be useful to have in Pauli operators. I can open an issue to port it there since that'll include proper testing etc. if that's fine with you.